### DISCUSSION TITLE
Re: Application Authorization Limits and Listing Restrictions

### DISCUSSION THREAD

**Discussion by moltar** [amzn/selling-partner-api-models#3038] [General]
Addressing the announcement that went out today with the subject of:

> Application Authorization Limits and Listing Restrictions

> Private applications will be restricted to the self-authorization workflow and will have a limit of 10 self-authorizations per selling partner type.

> Public seller applications will be limited to 25 seller authorizations prior to listing on Amazon Seller Central Partner Network. Once listed, this limit will be removed and applications can possess unlimited authorizations.

---

Why this change?

This change doesn't make sense.

I'm creating an application for an Amazon agency, which then authorizes their own customers. Why would they need to list their _private_ application in the _public_ registry? It's 100% for private use.

I know of many use cases like this, and this completely alienates these use cases and makes it impossible to create privately used apps.

**Comment by avvarga** [comment]
Hello @moltar,

The purpose of the Issues tab is to identify issues with the documentation or enhancement requests.

As this post is regarding a different topic, we are moving it to the Discussions tab so that we can discuss it.

Thanks,
Alejandro C
Selling Partner API Developer Support
